docs/blockchain/contracts/nft-contract/nft-actions/rmnftofr.a.experimental.md

```diff
@@ -0,0 +1,58 @@
+---
+title: 'rmnftofr.a'
+order: 39
+
+---
+
+# rmnftofr.a
+
+Cancel the buy offer made on the token
+
+## Technical Behavior
+
+Note that `rmnftofr.a` is used to cancel the offer made by `mknftofr.a` only.
+
+A buyer can cancel any of his offers at any time
+
+Once offer is expired, anyone can cancel the offer.
+
+When an offer is canceled, the offered price will be transferred from eosio.nftofr account back to buyer.
+
+## Action Parameters
+
+| Property Name | C++ Type | JavaScript Type | Definition                    |
+| ------------- | -------- | --------------- | ----------------------------- |
+| canceler      | name     | String          | Account who cancel an offer   |
```

Comment:
Account who cancel an offer
should be
 Account who cancels an offer

Reply:
updated